test/suite/test_config04.py

```diff
@@ -154,6 +153,47 @@ def test_eviction_bad2(self):
                                  'eviction_trigger=86'),
             "/eviction target must be lower than the eviction trigger/")
 
+    def test_eviction_absolute(self):
+        self.common_test('eviction_target=50MB,eviction_trigger=60MB,'
+            'eviction_dirty_target=20MB,eviction_dirty_trigger=15MB,'
+            'eviction_checkpoint_target=13MB')
+
+    def test_eviction_absolute_bad(self):
+        self.assertRaisesWithMessage(wiredtiger.WiredTigerError, lambda:
+            self.wiredtiger_open('.','create,eviction_target=70MB,'
+                                 'eviction_trigger=60MB'),
+            '/eviction target must be lower than the eviction trigger/')
```

Comment:
It'd be nice if there were a few more failure cases in here where inputs exceed other inputs that they aren't allowed to.
It would also be interesting to set a cache size to less than 1% via the absolute size, and verify that the setting is as expected.

Reply:
I would also like to see something similar to `common_cache_size_test` to verify that the settings are correct.